build/platform.simulator.mak

```diff
@@ -3,3 +3,5 @@ USE_LIBA = 0
 EXE = elf
 OS_WITH_ONBOARDING_APP ?= 0
 OS_WITH_SOFTWARE_UPDATE_PROMPT ?= 0
+CFLAGS = -fPIC
```

Comment:
Instead, simply use "SFLAGS" (which stands for shared flags, shared between C and C++).
You may also want to use -fPIE since we're building executables.